### PR TITLE
chore: add mint data

### PR DESCRIPTION
**Pull request by DavidMinarsch**

### CODE REVIEW

File: mints/mapping.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+token_id-"component_id"
+X-"skill/valory/apy_estimation_abci/0.1.0" - 1, 4, 14
```
Comment (DavidMinarsch) [inline mints/mapping.txt:2]: 1 and 4 should be removed as dependencies. One of them is not used in code, the other one is unnecessarily used.

File: mints/mapping.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+X-"skill/valory/apy_estimation_chained_abci/0.1.0" - 14, 16, 17, 22, 33, "skill/valory/apy_estimation_abci/0.1.0"
```
Comment (DavidMinarsch) [inline mints/mapping.txt:3]: Termination and transaction_settlement skills are missing in code.

File: mints/mapping.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+---
+token_id-"agent_id"
```
Comment (DavidMinarsch) [inline mints/mapping.txt:5]: Why is the ipfs connection and skill not listed as a dependency?

File: mints/mapping.txt (hunk 1 above)
Comment (Adamantios) [reply mints/mapping.txt:2]: There are the `abci` and the `http` protocols, correct? Why should one of them be removed?

File: mints/mapping.txt (hunk 3 above)
Comment (Adamantios) [reply mints/mapping.txt:5]: The IPFS connection and protocol are now listed as dependencies in #58.

File: mints/mapping.txt (hunk 1 above)
Comment (DavidMinarsch) [reply mints/mapping.txt:2]: They are pointlessly imported

**Pull request merged**